src/box/lua/config/utils/aboard.lua

```diff
@@ -92,6 +92,13 @@ local function aboard_drop_if(self, filter_f)
     end
 end
 
+-- Alter all alerts using the provided function.
+local function aboard_alter(self, alter_f)
+    for key, alert in pairs(self._alerts) do
+        alter_f(key, alert)
+    end
+end
```

Comment:
Nit: It doesn't alter the alerts on its own, so I would call it just `:each()`.

Reply:
Thank you for the review! Fixed.